src/pages/vereniging/besturen.jsx

```diff
@@ -13,57 +13,37 @@ const BoardPage = props => {
       <h2>{page.title}</h2>
       <Markdown>{page.content.content}</Markdown>
       <h3>Het huidig bestuur</h3>
-      <CurrentBoard>{getCurrentBoard(boards)}</CurrentBoard>
+      <Grid stretched stackable doubling columns={3}>
```

Comment:
I think it would be nice to have the current board card a little larger and centered.

Reply:
Yes I agree!

Reply:
And maybe also their names already displayed?

Reply:
Sure, but what is the extra effect of having a board page for them then

Reply:
Just consistency with the other boards. But I also like the way it is now :).